ctapipe/image/muon/muon_processor.py

```diff
@@ -230,7 +223,7 @@ def get_fov(self, tel_id):
 
     def get_pixel_width(self, tel_id):
         """Guesstimate fov radius for telescope with id `tel_id`"""
-        # memoize fov calculation
```

Comment:
The comment was regarding the wrong docstring and what is calculated (fov instead of pixel width)